### PR TITLE
MINOR: a simple benchmark for Streams

### PR DESCRIPTION
**Pull request by ymatsuda**
@guozhangwang @miguno

### CODE REVIEW

File: streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+import java.util.concurrent.CountDownLatch;
+import java.util.Properties;
+
+public class SimpleBenchmark {
+
+    private final String kafka;
+    private final String zookeeper;
+    private final File stateDir;
+
+    private static final String SOURCE_TOPIC = "simpleBenchmarkSourceTopic";
```
Comment (guozhangwang) [inline streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java:57]: Is there a script for preparing the data in the source topic?

EDIT: never mind.

File: streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
+    private static final String SINK_TOPIC = "simpleBenchmarkSinkTopic";
+
+    private static final long NUM_RECORDS = 10000000L;
+    private static final Long END_KEY = NUM_RECORDS - 1;
+    private static final int KEY_SIZE = 8;
+    private static final int VALUE_SIZE = 100;
+    private static final int RECORD_SIZE = KEY_SIZE + VALUE_SIZE;
```
Comment (guozhangwang) [inline streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java:64]: The actual throughput would be a bit larger when we counted the message metadata, but I think it does not matter for internal comparison here.

File: streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java (hunk 1 above)
Comment (ymatsuda) [inline streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java:57]: It is this benchmark program itself. It populates the source topic when measuring producer performance.

File: streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java (hunk 2 above)
Comment (ymatsuda) [inline streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java:64]: Metadata size is irrelevant.

**Comment by guozhangwang** [comment]
LGTM. Merged to trunk and 0.10.0.

**Comment by gwenshap** [comment]
Minor? really? :)

**Pull request closed without merging**